saltyrtc/server/events.py

```diff
@@ -0,0 +1,31 @@
+import collections
+from enum import (
```

Comment:
Please avoid `from` imports for external packages (unless nesting becomes too uncomfortable). I use this coding convention to be able to differentiate external and internal classes/functions easily in code without having to look at the imports.